consensus/broadcast/shardChainMessenger.go

```diff
@@ -231,6 +232,7 @@ func (scm *shardChainMessenger) PrepareBroadcastBlockDataValidator(
 	err = scm.delayedBlockBroadcaster.SetValidatorData(broadcastData)
 	if err != nil {
 		log.Error("shardChainMessenger.PrepareBroadcastBlockDataValidator", "error", err)
+		return
```

Comment:
this return is redundant

Reply:
I would keep it just for the case when the method would be extended by someone who would forgot to added back